internal/handler/handler.go

```diff
@@ -33,6 +33,81 @@ type userMgmtSvc struct {
 	logic logic.UserMgmtSvcLogicIer
 }
 
+func validatePass(pass string) *respModel.Response {
```

Comment:
Update this function to only return a `error` type.
For any failures/errors, the error coded message can be wrapped using `fmt.Errorf()` and returned as error.
At the caller of this function, check for error returned and return the error message in the `message` field using `err.Error()` and `BadRequest` status code.
In case no error is obtained, continue downstream flows with pass.

Reply:
done